console/src/test/java/org/georchestra/console/integration/UsersIT.java

```diff
@@ -184,6 +218,36 @@ private void setRole(String userName, String role1Name, String role2Name) throws
                 .andExpect(jsonPath("$.extractor").value(0))//
                 .andExpect(jsonPath("$.geodocs").value(0))//
                 .andExpect(jsonPath("$.ogcStats").value(0));
+
     }
 
+    @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+    @WithMockRandomUidUser
+    @DBUnit(qualifiedTableNames = true, dataTypeFactoryClass = PostgresExtendedDataTypeFactory.class)
+    @DataSet(executeScriptsBefore = "dbunit/geonetwork_ddl.sql", strategy = CLEAN_INSERT, value = { "dbunit/all.csv" })
+    public @Test void testDeleteUserAlsoDeletesGDPRAccountRecords() throws Exception {
+        createUser("user1");
+        deleteUser("user1")//
+                .andExpect(status().isOk())//
+                .andExpect(content().contentTypeCompatibleWith("application/json"))//
+                // .andDo(print())//
+                .andExpect(jsonPath("$.account").value("user1"))//
+                .andExpect(jsonPath("$.metadata").value(2))//
+                .andExpect(jsonPath("$.extractor").value(2))//
+                .andExpect(jsonPath("$.geodocs").value(3))//
+                .andExpect(jsonPath("$.ogcStats").value(3));
+
+        deleteUser("user1")//
+                .andExpect(status().isNotFound())//
+                .andExpect(content().contentTypeCompatibleWith("application/json"))//
+                .andDo(print())//
```

Comment:
:thinking: what will it print ?

Reply:
debugging aid to print how the mvc handled the request, inputs, outputs, controller used, etc. It's commented out now.